plugins/rest/rest.go

```diff
@@ -10,12 +10,13 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
-	"github.com/open-policy-agent/opa/internal/version"
 	"io"
 	"net/http"
 	"reflect"
 	"strings"
 
+	"github.com/open-policy-agent/opa/internal/version"
```

Comment:
Still a bit odd here

Reply:
Ah, you mean we should group them with the other open-policy-agent imports below? Happy to take care of that!

Reply:
Yup that's what I meant. There are a few more of these below, though...

Reply:
Hope I caught them all!